### DISCUSSION TITLE
g6 4.8版本升级到g6 @5.0.0-beta.21版本 出现loader无法解析指定文件的报错

### DISCUSSION THREAD

**Discussion by Cersei2005** [antvis/G6#5089] [Q&A] [answered]
error  in ./node_modules/@antv/graphlib/esm/graph.js

Module parse failed: Unexpected token (5:12)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| import { doBFS, doDFS } from './utils/traverse';
| export class Graph extends EventEmitter {
>     nodeMap = new Map();
|     edgeMap = new Map();
|     inEdgesMap = new Map();

 @ ./node_modules/@antv/graphlib/esm/index.js 2:0-24 2:0-24
 @ ./node_modules/@antv/g6/esm/runtime/controller/data.js
 @ ./node_modules/@antv/g6/esm/runtime/controller/index.js
 @ ./node_modules/@antv/g6/esm/runtime/graph.js
 @ ./node_modules/@antv/g6/esm/index.js
 @ ./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/views/logicalTopo/Topo/pages/testTopo.vue?vue&type=script&lang=js&
 @ ./src/views/logicalTopo/Topo/pages/testTopo.vue?vue&type=script&lang=js&
 @ ./src/views/logicalTopo/Topo/pages/testTopo.vue
 @ ./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./src/views/login/index.vue?vue&type=script&lang=js&

**Comment by Aarebecca** [comment, marked as answer]
根据错误信息，应该是你的构建工具缺少必要的 loader 导致的

**Comment by nihaoll** [comment]
so,do i need what kind of loader
